bundle/manifests/multiarch.openshift.io_clusterpodplacementconfigs.yaml

```
@@ -78,16 +77,14 @@ spec:
                   label "multiarch.openshift.io/exclude-pod-placement" exists.
                 properties:
                   matchExpressions:
-                    description: matchExpressions is a list of label selector requirements.
-                      The requirements are ANDed.
+                    description: matchExpressions is a list of label selector requirements. The requirements are ANDed.
                     items:
                       description: |-
                         A label selector requirement is a selector that contains values, a key, and an operator that
                         relates the key and values.
                       properties:
                         key:
-                          description: key is the label key that the selector applies
-                            to.
+                          description: key is the label key that the selector applies to.
```

```suggestion
                          description: The key that is used as part of the selector
```
Not an action necessarily for you, however the above may need some word smithing beyond my ability.

---

To change in the golang comment, but thanks @prb112

---

Right...